### ISSUE TITLE
`indentation_width` should exclude indentation within multiline strings

### ISSUE THREAD

**Issue by robertlacroix** [realm/SwiftLint#4248]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

`indentation_width` probably shouldn't flag content in multiline strings, as it's likely not Swift code and might follow different indentation norms.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint test.swift --config indent.yml
Linting Swift files at paths test.swift
Linting 'test.swift' (1/1)
test.swift:3:1: warning: Indentation Width Violation: Code should be indented using one tab or 4 spaces. (indentation_width)
Done linting! Found 1 violation, 0 serious in 1 file.

$ cat test.swift
let jsonFixture = """
{
  "test": true
}
"""
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.49.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)? building from source
* Paste your configuration file:

```yml
opt_in_rules:
  indentation_width
```
